### ISSUE TITLE
`show-config` fails when verifying the nix-config when using nix-master

### ISSUE THREAD

**Issue by NobbZ** [LnL7/nix-darwin#823]
A user of the unofficial nix Discord stumbled upon this.

They decided to simply use a tagged nix for now, and I do not have any more detailed logs about the issue.

Though in general I assume the issue is basically the same as in https://github.com/NixOS/nixpkgs/issues/271146

**Comment by nrabulinski** [comment]
Yes, it’s exactly the same issue https://github.com/LnL7/nix-darwin/blob/4b9b83d5a92e8c1fbfd8eb27eda375908c11ec4d/modules/nix/default.nix#L71-L74
